### PR TITLE
rewrite java client demo

### PR DESCRIPTION
**Pull request by ljcui**
- Make code more friendly for java engineers

### CODE REVIEW

File: demo/movie/import_for_java_demo.json [hunk 1]
```diff
@@ -0,0 +1,316 @@
+{
```
Comment (knightast) [inline demo/movie/import_for_java_demo.json:1]: 为啥要给java单独准备一个json

Comment (ljcui) [reply demo/movie/import_for_java_demo.json:1]: > 为啥要给java单独准备一个json

json文件files部分csv文件的相对路径不一样，需要改下。

**Comment by jiazhenjiang** [comment]
looks good to me

**Review by jiazhenjiang — APPROVED** [review]

**Comment by knightast** [comment]
the doc directory should be:
doc/zh-CN/...
doc/en-US/...

Please put the doc in the right directory, @KingSkyLi will do the rest of English translation in one month.

**Pull request merged**